iamclient/iamclient.go

```diff
@@ -0,0 +1,143 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Maybe there is no sense to keep a connection to IAM. We need this connection only on startup to get key, cert URL.

Reply:
Done